examples/JointCases/joint_cases_callback.py

```diff
@@ -0,0 +1,22 @@
+from elastica.callback_functions import CallBackBaseClass
+
+
+class JointCasesCallback(CallBackBaseClass):
```

Comment:
You can make alias `JointCasesCallback = MyCallback` instead to reduce code duplication. We want to minimize extra modules within the example cases.

Reply:
Thank your suggestion. Fixed in https://github.com/GazzolaLab/PyElastica/pull/135/commits/5b379506e3dda6c2344c945d85759955737108c2

Reply:
Please remove this file, I don't think it is used.

Reply:
Addressed in https://github.com/GazzolaLab/PyElastica/pull/135/commits/6d360f6352393a911ce0aa7a4425defd1fa5dee4